code/lightna/magento-os-frontend/js/product/Options.js

```diff
@@ -1,13 +1,24 @@
-import { $, $$ } from '../lib/utils';
+import { $, $$, getBlockHtml } from '../lib/utils';
+import { UserInput } from '../lib/UserInput';
 
 export class ProductOptions {
+    cjs = '.cjs-product-options';
+    block = '.body.main.content.main.cta.add-to-cart.options';
+
     constructor() {
-        this.jsc = $('.cjs-product-options');
+        this.init();
+    }
+
+    init() {
+        this.jsc = $(this.cjs);
```

Comment:
in my MR you told me init not needed if we have constructor 😀

Reply:
in this case it's called in another method, however we can use new ProductOptions() instead, what do prefer?

Reply:
it could also help to find whre it's used in js if you see block id

Reply:
creating a new instance doesn't sound like a good idea, so better leave init

Reply:
btw, makes sense to rename it as well
```suggestion
        this.cjs = $(this.cjs);
```

Reply:
solved